src/crawlers/cheerio_crawler.js

```diff
@@ -333,13 +328,13 @@ class CheerioCrawler {
      */
     async _handleRequestFunction({ request, autoscaledPool }) {
         if (this.prepareRequestFunction) await this.prepareRequestFunction({ request });
-        const { dom, response } = await addTimeoutToPromise(
+        const { dom, responseStream: response } = await addTimeoutToPromise(
             this._requestFunction({ request }),
             this.requestTimeoutMillis,
             'CheerioCrawler: requestFunction timed out.',
         );
 
-        request.loadedUrl = response.request.uri.href;
+        request.loadedUrl = response.url; // @TODO:  add test for redirects
```

Comment:
Is this TODO done?

Reply:
Yes tested on the google shopping seller urls.